_security/access-control/api.md

```diff
@@ -1297,6 +1297,85 @@ PATCH _plugins/_security/api/securityconfig
 }
 ```
 
+### Check configuration upgrade
+Introduced 2.13
+{: .label .label-purple }
+
+Checks the configuration bundled with the Security plugin against the existing security configuration, returns if an upgrade can be performed and what resources would be updated.
+
+#### Request
+
+```json
+GET _plugins/_security/api/_upgrade_check
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+```json
+{
+  "status" : "OK",
+  "upgradeAvailable" : true,
+  "upgradeActions" : {
+    "roles" : {
+      "add" : [ "flow_framework_full_access" ]
+    }
+  }
+}
+```
+
+#### Response fields
+
+| Field    | Data type  | Description                   |
+|:---------|:-----------|:------------------------------|
+| `upgradeAvailable`   | Boolean     | True when an upgrade can be performed.  |
+| `upgradeActions`  | Object     | The security objects that would be modified by an upgrade.  |
+
+### Perform configuration upgrade
+Introduced 2.13
+{: .label .label-purple }
+
+Adds and updates resources on the existing security configuration from the configuration bundled with the Security plugin.
+
+These bundled configuration files can be found in the `opensearch-project/security/config` directory.  Default configuration files are updated on OpenSearch upgrade, whereas the cluster configuration is only updated by the cluster operators. This request used to bridge these scenarios.
+
+#### Request
+
+```json
+POST _plugins/_security/api/_upgrade_check
+{
+  "configs": ["roles"]
+}
+```
+{% include copy-curl.html %}
+
+#### Request fields
+
+| Field           | Data type  | Description                                                                                                       | Required |
+|:----------------|:-----------|:------------------------------------------------------------------------------------------------------------------|:---------|
+| configs              | Array     | Specifies the configurations to be upgraded. This field can include any combination of the following values: `actiongroups`,`allowlist`, `audit`, `internalusers`, `nodesdn`, `roles`, `rolesmappings`, `tenants`.  If absent, all supported configurations are included.  | No      |
```

Comment:
Some of these descriptions are pretty long, have you tested how they are rendered?

Should we try to render this list like a bullet-point list?

Reply:
I've looked around, I'm going to update this like another example from within this site: https://opensearch.org/docs/latest/data-prepper/managing-data-prepper/peer-forwarder/#configuration

Line wrapping is on by default, but I'll see how these render out

Reply:
![image](https://github.com/opensearch-project/documentation-website/assets/2754967/d3a6e4ab-af48-42ef-9228-b8c3cebf66ee)